src/main/java/nextstep/calculator/selector/SeparatorStrategySelector.java

```diff
@@ -0,0 +1,29 @@
+package nextstep.calculator.selector;
+
+import nextstep.calculator.domain.BaseSeparator;
+import nextstep.calculator.domain.CustomSeparator;
+import nextstep.calculator.domain.SeparatorStrategy;
+import org.apache.commons.lang3.StringUtils;
+
+
+public class SeparatorStrategySelector {
+
+    private String inputText;
+
+    public SeparatorStrategySelector(String inputText) {
+        this.inputText = inputText;
+    }
+
+    public SeparatorStrategy selectSeparatorStrategy() {
```

Comment:
```suggestion
    public SeparatorStrategy selectSeparatorStrategy(String inputText) {
```

Calculator와 동일한 내용일 것 같은데, 매번 객체를 생성하는 설계를 하신 이유가 있으실까요!?

Reply:
앗 질문에 대한 의도를 정확하게 파악하지 못하였습니다! 가능하시다면 조금 자세히 설명 가능하실까요?!
SeparatorStrategySelector 클래스 객체를 생성하고, 이후에 inputText 값에 따라 SeparatorStrategy 구현체 클래스 객체를 생성하는 것을 분리했다는 것에 대한 질문일까요?!

그 질문이 맞으시다면, 굳이 SeparatorStrategySelector 객체는 생성할 필요는 없을 것 같네요 😆